### PR TITLE
ref(uninstall): refactor uninstall to use kubectl pkg

### PR DESCRIPTION
**Pull request by adamreese**

### CODE REVIEW

**Comment by michelleN** [comment]
@adamreese This looks good. How do you feel about throwing in some tests here?

File: helm.go [hunk 1]
```diff
@@ -185,8 +185,12 @@ This will not alter the charts in your workspace.
 			ArgsUsage: "[chart-name...]",
 			Action: func(c *cli.Context) {
 				minArgs(c, 1, "uninstall")
+
+				// Todo: add dry-run flag
+				client := kubectl.Client
```
Comment (technosophos) [inline helm.go]: is there a ticket for this?

**Comment by arschles** [comment]
@adamreese can you write some tests for `uninstallKind`? that has non-trivial logic added

**Comment by adamreese** [comment]
@michelleN @arschles working up some tests

File: helm.go (hunk 1 above)
Comment (adamreese) [inline helm.go]: No, I should update my Todo statement to read 'Create a ticket to add `dry-run` flag to uninstall'

**Pull request merged**